docs/SPEC.md

```diff
@@ -1198,6 +1198,22 @@ Example:
     from(bucket:"telegraf/autogen")
     from(bucketID:"0261d8287f4d6000")
 
+#### Buckets
+
+Buckets is a type of data source that retrieves a list of buckets that the caller is authorized to access.  
+It takes no input parameters and produces an output table with the following columns: 
+
+* name: the name of the bucket
+* id: the internal ID of the bucket
+* organization: the organization this bucket belongs to
+* organizationID: the internal ID of the organization
+* Retention Policy: the name of the retention policy, if present
```

Comment:
Do retention policies have names? I didn't think they did.

Reply:
in what context?  in v1.X they do: 

> show retention policies
name    duration shardGroupDuration replicaN default
----    -------- ------------------ -------- -------
autogen 0s       168h0m0s           1        true